src/components/app/App.vue

```diff
@@ -1,19 +1,17 @@
-<template>
-  <div id="app">
-    <header>
-      <navbar></navbar>
-    </header>
-    <main>
-      <transition name="fade" mode="out-in" appear>
-        <router-view></router-view>
-      </transition>
-    </main>
-    <footer>
-      <span class="pull-right">
-        {{ $t('general.total_registrations_label', { count: usersTotalCount }) }}
-      </span>
-    </footer>
-  </div>
+<template lang="pug">
+  #app
+    header
+      navbar
+    main
+      transition(
+        name="fade"
+        mode="out-in"
+        appear
+      )
+        router-view
```

Comment:
@yak80, it depends on situation, but if children element is only one, it's recommended to use it like:

```
main
  transition(
    name="fade"
    mode="out-in"
    appear
  ): router-view
```